gallery/src/pages/lovelace/thermostat-card.ts

```diff
@@ -57,6 +69,23 @@ const CONFIGS = [
   entity: climate.nest
     `,
   },
+  {
+    heading: "Fan only example",
+    config: `
+- type: thermostat
+  entity: climate.sensibo
+  features:
+    - type: climate-hvac-modes
+      hvac_modes:
+        - fan_only
+        - 'off'
+    - type: climate-fan-modes
+      style: icons
+      fan_modes:
```

Comment:
also for `fan` platform ?

Reply:
Indeed, at the time of comment I did not know what the `fan` platform supports so I commented to check that later.

That's why, but if you say so I believe you 😉